src/mol-model-props/computed/accessible-surface-area/shrake-rupley.ts

```diff
@@ -102,31 +103,11 @@ namespace AccessibleSurfaceArea {
 
     export function getValue(location: StructureElement.Location, accessibleSurfaceArea: AccessibleSurfaceArea) {
         const { area, serialResidueIndex } = accessibleSurfaceArea;
-        const rSI = serialResidueIndex[indexOf(location.structure.root.serialMapping.elementIndices, location.element)];
+        const rSI = serialResidueIndex[SortedArray.indexOf(location.structure.root.serialMapping.elementIndices as unknown as SortedArray, location.element)];
```

Comment:
You can use SortedArray.ofSortedArray() instead of manually casting

Reply:
Done with f930e3d.
Feel free to merge at any time and thanks again for the effective review!